tests/test_volume.py

```diff
@@ -291,6 +291,32 @@ def test_save_load(vols_1):
         assert np.allclose(vols_1, vols_loaded_single)
         assert isinstance(vols_loaded_double, Volume)
         assert np.allclose(vols_1, vols_loaded_double)
+        assert vols_loaded_single.pixel_size is None, "Pixel size should be None"
+        assert vols_loaded_double.pixel_size is None, "Pixel size should be None"
+
+
+def test_save_load_pixel_size(vols_2):
+    # Create a tmpdir in a context. It will be cleaned up on exit.
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # Save the Volume object into an MRC files
+        mrcs_filepath = os.path.join(tmpdir, "test.mrc")
+        vols_2.save(mrcs_filepath)
+
+        # Load saved MRC file as a Volume of dtypes single and double.
+        vols_loaded_single = Volume.load(mrcs_filepath, dtype=np.float32)
+        vols_loaded_double = Volume.load(mrcs_filepath, dtype=np.float64)
+
+        # Confirm the pixel size is loaded
+        np.testing.assert_approx_equal(
+            vols_loaded_single.pixel_size,
+            vols_2.pixel_size,
+            err_msg="Incorrect pixel size in singles.",
+        )
```

Comment:
Maybe we should check that `vols_2.pixel_size` is correct before this comparison. This is done sort of implicitly through the synthetic volume test, but maybe doing it explicitly here for `Volume`s would be good.

Reply:
Sure, added a direct test.